src/osp/Active/SysNewton.cpp

```diff
@@ -47,20 +49,21 @@ void cb_force_torque(const NewtonBody* pBody, dFloat timestep, int threadIndex)
     // Get Scene from Newton World
     ActiveScene* scene = static_cast<ActiveScene*>(
                 NewtonWorldGetUserData(NewtonBodyGetWorld(pBody)));
+    auto &rReg = scene->get_registry();
 
     // Get ACompNwtBody. Every NewtonBody created here is associated with an
     // entity that contains one.
     auto *pBodyComp = static_cast<ACompNwtBody*>(NewtonBodyGetUserData(pBody));
     auto &rTransformComp = scene->reg_get<ACompTransform>(pBodyComp->m_entity);
 
     // Check if transform has been set externally
-    if (rTransformComp.m_transformDirty)
+    auto &tfMutable = scene->reg_get<ACompTransformMutable>(pBodyComp->m_entity);
+    if (tfMutable.m_dirty)
```

Comment:
this is a nice pattern, if you're into it.
```cpp
if(std::exchange(tfMutable.m_dirty, false))
{
    NewtonBodySetMatrix(...);
}
```

Reply:
I'll keep this in mind. File has been moved in #112 however.